src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -42,7 +41,6 @@ const FORMNAME_CLASSNAME_MAPPINGS = {
     [AUTHENTICATOR_KEY.ON_PREM]: 'enroll-onprem',
     [AUTHENTICATOR_KEY.RSA]: 'enroll-rsa',
     [AUTHENTICATOR_KEY.DUO]: 'enroll-duo',
-    [AUTHENTICATOR_KEY.SYMANTEC_VIP]: 'enroll-symantec',
```

Comment:
This is correct.. why was this removed?

Reply:
Accidental removal on my end - will fix!